crates/sovereign-sdk/full-node/sov-ledger-rpc/src/server.rs

```diff
@@ -240,48 +135,14 @@ where
         },
     )?;
 
-    rpc.register_subscription(
-        "ledger_subscribeSlots",
-        "ledger_slotProcessed",
-        "ledger_unsubscribeSlots",
-        |_, pending_subscription, db| async move {
-            // Register with the ledgerDB to receive callbacks
-            let mut rx = db
-                .subscribe_slots()
-                .map_err(|e| to_jsonrpsee_error_object(LEDGER_RPC_ERROR, e))?;
-
-            // Accept the subscription. This message is sent immediately
-            let subscription = pending_subscription.accept().await?;
-            let closed = subscription.closed();
-            futures::pin_mut!(closed);
-
-            // This loop continues running until the subscription ends.
-            loop {
-                let next_msg = rx.recv();
-                futures::pin_mut!(next_msg);
-                match futures::future::select(closed, next_msg).await {
-                    // If the subscription closed, we're done
-                    Either::Left(_) => break Ok(()),
-                    // Otherwise, we need to send the message
-                    Either::Right((outcome, channel_closing_future)) => {
-                        let msg = SubscriptionMessage::from_json(&outcome?)?;
-                        // Sending only fails if the subscriber has canceled, so we can stop sending messages
-                        if subscription.send(msg).await.is_err() {
-                            break Ok(());
-                        }
-                        closed = channel_closing_future;
-                    }
-                }
-            }
-        },
-    )?;
-
     Ok(rpc)
 }
 
+#[allow(dead_code)]
```

Comment:
Why?

Reply:
the fields inside it give unused warning